src/main/java/service/operation/BalanceOperation.java

```diff
@@ -0,0 +1,19 @@
+package service.operation;
+
+import data.db.Storage;
+import java.util.Map;
+import model.FruitTransaction;
+
+public class BalanceOperation implements OperationHandler {
+
+    @Override
+    public void operation(FruitTransaction.Operation operation, FruitTransaction fruitTransaction) {
```

Comment:
The method signature 'public void operation(FruitTransaction.Operation operation, FruitTransaction fruitTransaction)' should be revised. Since the BalanceOperation is focused on a single task, there is no need to pass the 'operation' parameter here. The operation type is implied by the class itself.